### ISSUE TITLE
Bug sidebar menu

### ISSUE THREAD

**Issue by ganjarsetia** [puikinsh/Adminator-admin-dashboard#292]
<h1>Bug report on sidebar menu</h1>

## Current behavior

I found bug on sidebar menu. It wont collapse on some screen size.

Steps to reproduce:

1. open https://colorlib.com/polygon/adminator/index.html in laptop size.
2. Click hamburger menu to make sidebar menu collapse.
3. Open Chrome Dev Tools and click [Toggle device toolbar ](https://uploads.sitepoint.com/wp-content/uploads/2017/01/1485343931chrome-mobile-emulation-1-enable.png)
4. In [Dimensions](https://wd.imgix.net/image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ldhUUNnixCCAmbF7cQrk.png?auto=format&w=964) choose "Nest Hub" or screen with width 1024px.
5. And it will show like this ![image](https://user-images.githubusercontent.com/3258090/186430781-6d0b2da5-bb99-4b24-9c1b-b1f88557f5f5.png)

## Expected behavior 

The sidebar should be collapsed. Like this ![image](https://user-images.githubusercontent.com/3258090/186432334-f162cbb2-0c30-4471-863b-0e9d96b52977.png)

# Environment
Platform: Linux Pop OS 20
Browser: Brave

**Comment by jepperaskdk** [comment]
Cool, thanks for submitting this. It does look like the CSS class `.is-collapsed` inverts the collapsing functionality, depending on screen size - maybe by design.

**Comment by ganjarsetia** [comment]
> Cool, thanks for submitting this. It does look like the CSS class `.is-collapsed` inverts the collapsing functionality, depending on screen size - maybe by design.

Yeah its inverts the functionality. So can it be fixed or any way how to fix it?
